talknet/demoTalkNet.py

```
@@ -1,726 +1,726 @@
+min_face_size = 1 # Minimum face size in pixels
+crop_scale = 0.40 # Scale bounding box
+start = 0 # The start time of the video
+duration = 0 # The duration of the video, when set as 0, will extract the whole video
+pyaviPath = os.path.join(save_path, 'pyavi')
+pyframesPath = os.path.join(save_path, 'pyframes')
+pyworkPath = os.path.join(save_path, 'pywork')
+pycropPath = os.path.join(save_path, 'pycrop')
+videoFilePath = os.path.join(pyaviPath, 'video.avi')
+audioFilePath = os.path.join(pyaviPath, 'audio.wav')
+
+if os.path.isfile(pretrained_model_path) == False: # Download the pretrained model
```

Ruff (0.13.3)

Avoid equality comparisons to `False`; use `not os.path.isfile(pretrained_model_path):` for false checks

Replace with `not os.path.isfile(pretrained_model_path)`

(E712)